### ISSUE TITLE
Export logic from Maatwebsite ExportClass

### ISSUE THREAD

**Issue by xlyxer** [pxlrbt/filament-excel#59]
Is there any possibility to make an ExportAction via Maatwebsite custom class as it was in previous versions via WithExportable trait?

**Comment by pxlrbt** [comment]
Yes. The `ExcelExport` class implements the Maatwebsite Concerns, so you can still work on that base layer. The class is just an abstraction layer for easier configuration.

```php
ExportAction::make()->exports([
    YourMaatwebsiteExportClass::make('table'),
])

**Comment by pxlrbt** [comment]
Closing as stale.
